src/api/entities/Asset/NonFungible/Nft.ts

```diff
@@ -18,8 +18,6 @@ export type NftUniqueIdentifiers = {
  * Class used to manage Nft functionality
  */
 export class Nft extends Entity<NftUniqueIdentifiers, string> {
-  public ticker: string;
```

Comment:
I felt this was a redundant since there is a collection instance. I didn't mark it as a breaking change since NFTs are still in alpha only